cypress/support/commands.js

```diff
@@ -85,3 +88,28 @@ Cypress.Commands.add("visitSite",(site ,clickOn)=>{
     cy.contains(clickOn).click()
     cy.url().should("include", site)
 })
+
+//Parameter
+//saveSlot string: Expample: "swot-1" for a first step save for SWOT, which slot has what save look into saveDummyData.json in fixtures
+//name string: name for the save
+//tool_id int : Which tool, 2 for SWOT-Analysis
+//owner_id int: optional: standard is 0 for max@test.test
+Cypress.Commands.add('CreateSave',(saveSlot,name,tool_id,owner_id = 1)=>{
+    
+    cy.fixture("saveDummyData.json").then(function (SaveData)
```

Comment:
the dummy data is currently forced to be in one file for **all** tests. This is could get very messy very fast. I would suggest to move every save data in its own file.

I would suggest to create two files, one named `swot-1.json` and one `swot-2.json`.

Reply:
Maybe put these files inside its own folder inside the fixtures folder